gh-pr-context.py

```
@@ -1,22 +1,494 @@
+        key=lambda x: x.get("name", ""),
+    )
+
+    if not failed:
+        return
+
+    for run in failed:
+        job_id = run["id"]
+        name = run["name"]
+        log_content = ""
+        try:
+            args = _gh_cmd() + ["api", f"repos/{owner_repo}/actions/jobs/{job_id}/logs"]
```

Ruff (0.15.12)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)